app/serializers/tag.js

```diff
@@ -29,7 +29,27 @@ export default DS.JSONSerializer.extend({
   // Modify "tag" objects to become "response" objects.
   // Only used for sending data, such as: tag.save() .
   serialize: function(snapshot) {
+
+    var member = snapshot.belongsTo('member'), // Get the member that the tag belongs to
+      chapter_id = snapshot.attr('chapterId'),
+      progress = member.attr('progresses').filterBy('chapter_id', chapter_id).objectAt(0),
+      started,
+      ended;
+
+    if (progress.first_tag_sent === true) {
+      started = true;
+      delete progress.first_tag_sent; // Remove flag, because it is only to be sent once
+    }
+
+    // For Solarium use only; This does not affect summer-app in any way
+    // Only send 'ended=true' flag if the member is in a state of 'completed' or 'unqualified'
+    if (progress.status === 'completed' || progress.status === 'unqualified') {
+      ended = true;
```

Comment:
If the user goes back into the survey after successfully completing it, they are still marked as `ended`. Not a bug, but it allows for many `ended` flags to be sent.